lib/vrl/parser/src/parser.lalrpop

```diff
@@ -0,0 +1,511 @@
+use std::str::FromStr;
+use crate::ast::*;
+use crate::lex::*;
+use diagnostic::span;
+use std::collections::BTreeMap;
+use lalrpop_util::ErrorRecovery;
+use ordered_float::NotNan;
+
+grammar<'err, 'input>(input: &'input str);
```

Comment:
Is the `'err` needed here? I don't see it used anywhere.

Reply:
No, left-over from older implementation.